app/services/api/v1/ecf_participants/index.rb

```diff
@@ -104,7 +104,8 @@ def induction_record_ids_with_deduped_profiles
             .where(id: induction_record_ids_with_deduped_induction_records)
             .joins(participant_profile: [:participant_identity])
             .select("DISTINCT ON (participant_profiles.participant_identity_id) participant_identity_id, induction_records.training_status, participant_profiles.created_at, induction_records.id")
-            .order("participant_profiles.participant_identity_id", "induction_records.training_status ASC", "participant_profiles.created_at DESC")
+            .order("participant_profiles.participant_identity_id", "induction_records.training_status
```

Comment:
you've changed this class but it is no longer invoked. do you therefore now want to to delete this class or revert the changes back if you do not wish to change it?

Reply:
actually it is used in `serialized_response` but that goes back to my other comment of why are choosing to use 2 different serializers?